Wikipedia/Code/TabBarDelegate.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  TabBarDelegate.swift
+//  Wikipedia
+//
+//  Created by Matt Cl on 4/13/20.
+//  Copyright © 2020 Wikimedia Foundation. All rights reserved.
+//
+
+import Foundation
+
+@objc public protocol AppTabBarDelegate {
```

Comment:
I know the codebase isn't consistent with this but it's good to add a WMF prefix for obj-c:

```
@objc(WMFAppTabBarDelegate)
protocol AppTabBarDelegate {
```

Also I think you can remove the `public` and I would rename the file to AppTabBarDelegate.swift.

Reply:
Thanks for those - they've now been updated. As you could probably guess, I renamed the class after I made the file and forgot to update it. Thanks for catching that.

Reply:
I do it all the time 😁 LGTM, thanks!